spec/models/requested_topic_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'rails_helper'
+
+RSpec.describe RequestedTopic, type: :model do
+  describe 'name' do
+    subject { described_class.new(requested_topic) }
+    context 'when valid' do
```

Comment:
I believe we added shoulda matchers to the project. This whole spec file can be:

```ruby
require "rails_helper"

RSpec.describe RequestedTopic, type: :model do
  it { is_expected.to validate_presence_of(:name) }
end
```